core/src/main/java/com/graphhopper/util/PathMerger.java

```diff
@@ -90,7 +90,7 @@ public void doWork(PathWrapper altRsp, List<Path> paths, Translation tr) {
             fullDistance += path.getDistance();
             fullWeight += path.getWeight();
             if (enableInstructions) {
-                InstructionList il = path.calcInstructions(tr);
+                InstructionList il = path.calcInstructions(tr, origPoints);
```

Comment:
A better name now for origPoints might be origPointCount (?) as I initially thought it is the full PointList that must be injected into every InstructionList ...